Makefile

```diff
@@ -9,7 +9,7 @@ help: ## Display this help menu
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
```

Comment:
I use `phpstorm` and the plugin doesn't recongize this syntax https://github.com/kropp/intellij-makefile/issues/12#issuecomment-327003689. Is it possible to modify with:

```
ifndef ENV
	ENV=prod
endif
```

or 

```
ENV?=prod
```

Reply:
I will make a change this evening